storage/storage_transfer/transfer_check.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+
+import create_client
+
+# Edit these values with desired parameters
+PROJECT_ID = 'storage-sfan-test1'
```

Comment:
The motivation is to allow everyone run the test suite, when they want to contribute to this repo.